backend/tests/test_authorization.py

```
@@ -0,0 +1,282 @@
+        response = client.post('/api/auth/login', json=login_data)
+        user1_token = response.get_json()['access_token']
+        user1_headers = {'Authorization': f'Bearer {user1_token}'}
+        
+        # Login as second user
+        login_data = {
+            'employee_number': 'OTHER001',
+            'password': 'otherpass123'
+        }
+        response = client.post('/api/auth/login', json=login_data)
+        user2_token = response.get_json()['access_token']
+        user2_headers = {'Authorization': f'Bearer {user2_token}'}
```

Ruff (0.11.9)

Local variable `user2_headers` is assigned to but never used

Remove assignment to unused variable `user2_headers`

(F841)